Bottom: c8b231903bca4b8e54aee99d692d359afc88693d
Top:    e9b8a9b892a2a7f1163da61c6fee19baef6bfcda
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 21:22:21 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index b4a49e3..5435f41 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -3839,8 +3839,8 @@ hid_lesstif_init ()
   memset (&lesstif_hid, 0, sizeof (HID));
 
   common_draw_helpers_init (&lesstif_hid);
-  lesstif_hid.struct_size             = sizeof (HID);
 
+  lesstif_hid.struct_size             = sizeof (HID);
   lesstif_hid.name                    = "lesstif";
   lesstif_hid.description             = "LessTif - a Motif clone for X/Unix";
   lesstif_hid.gui                     = 1;
